frontend/src/services/templateService.ts

```diff
@@ -78,6 +82,19 @@ export const updateServingRuntimeTemplateBackend = (
             path: '/objects/0',
             value: servingRuntime,
           },
+          existingTemplate.metadata.annotations
+            ? {
+                op: 'replace',
+                path: '/metadata/annotations/opendatahub.io~1modelServingSupport',
+                value: JSON.stringify(platforms),
+              }
+            : {
+                op: 'add',
+                path: '/metadata/annotations',
+                value: {
+                  'opendatahub.io/modelServingSupport': JSON.stringify(platforms),
+                },
+              },
```

Comment:
Dupe code. `frontend/src/api/k8s/templates.ts` -- we should resolve this... it'll be a maintenance annoyance.